ReactNativeTemplate/ios/Podfile

```diff
@@ -27,6 +27,9 @@ pod 'React', :path => '../node_modules/react-native', :subspecs => [
     'RCTLinkingIOS'
 ]
 
+# NB: get rid of next line once FMDB works with xcode 9
+pod 'FMDB', :git => 'https://github.com/forcedotcom/fmdb', :branch => '2.7.2_xcode9' 
```

Comment:
Exactly, so no impact to internal teams or cordova plugin, but for cocoapods consumers of the SDK it will be treated like other dependencies.